notification-bot/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "apibara-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon --exec ts-node src/index.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
License of the whole governance project is MIT currently, but it will be changed to Apache 2.0 soon, so please use Apache 2.0 here.